homeassistant/components/acer_projector/switch.py

```diff
@@ -79,10 +81,10 @@ def __init__(
             port=serial_port, timeout=timeout, write_timeout=write_timeout
         )
         self._serial_port = serial_port
-        self._name = name
+        self._attr_name = name
         self._state = False
         self._available = False
-        self._attributes = {
+        self._attr_attributes = {
```

Comment:
```suggestion
        self._attr_extra_state_attributes = {
```
This still needs to be named `_attr_extra_state_attributes` it is only needs to be done inside init so that it is not common to all the class instances.